component/src/test/java/io/siddhi/extension/io/nats/sink/NATSSinkTestCase.java

```diff
@@ -1,288 +1,120 @@
 /*
- * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Same,

Reply:
Fixed in https://github.com/siddhi-io/siddhi-io-nats/pull/37/commits/867fce4dac46cc0371b9dd2dd32807f83c541684